rotkehlchen/tests/exchanges/test_bybit.py

```diff
@@ -52,6 +53,12 @@ def _mock_authenticated_query(path: str, options: dict[str, Any] | None = None)
                 },
             }
 
+        if (
```

Comment:
This is emulating the error you got from bybit? Can you write a comment here why?